classroom.rb

```diff
@@ -0,0 +1,15 @@
+require_relative './person'
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students.push(student)
+    student.class1 = self
```

Comment:
- [ ] The instance variable in the Student class is `classroom`, not `class1`. Please correct that. 
```rb
student.classroom = self
```